### ISSUE TITLE
train problem

### ISSUE THREAD

**Issue by ujgygy656565** [ChiWeiHsiao/DeepVO_PyTorch#21]
Hello, I am a novice. I have two questions. I set resume to false. I want to train it from the beginning. But when I trained 100 to 200 epochs, the results of the tests and visualize are very poor. I haven't changed your parameters and code. What's the reason? The second question is, the img-means and stds in params are calculated by preprocess? Thank you very much.

**Comment by ujgygy656565** [comment]
I set resume to false. According to the parameters given in params, I trained to 100 epoch and 250 epoch, and tested them separately. But the effect of visualize is very poor. Is epoch not enough or where can be adjusted? Can you teach me?

**Comment by ujgygy656565** [comment]
Why don't I change the parameters and train from scratch to 400 epoch and the result is like this. Thank you. 
![image](https://user-images.githubusercontent.com/46241022/57610461-7126d480-75a3-11e9-9312-764b553c2dae.png)
![image](https://user-images.githubusercontent.com/46241022/57610500-856ad180-75a3-11e9-8d65-00d859048d26.png)
![image](https://user-images.githubusercontent.com/46241022/57610517-8bf94900-75a3-11e9-8314-8037d403af1b.png)

**Comment by alexart13** [comment]
@ujgygy656565 Did you use a pretrained FlowNet?

**Comment by ujgygy656565** [comment]
> @ujgygy656565 Did you use a pretrained FlowNet?

yes，I changed pretained flownet in params to. / pretrained / flownets_bn_EPE2.459. pth. tar. What may be my problem? Is the label relative position and posture in training? In which file is it expressed? Thank you.

**Comment by alexart13** [comment]
try flownets_EPE1.951.pth.tar. Also it seems that your NN is undertrained.

**Comment by ujgygy656565** [comment]
> try flownets_EPE1.951.pth.tar. Also it seems that your NN is undertrained.

Thank you. I've got the normal trajectory now. How to calculate the evaluation index of KITTI SLAM?

**Comment by DexterFixxor** [comment]
@ujgygy656565 Hello, what did you do to solve your problem? It appears that I have similar situation. While training, my RMSE is very small for both train and valid set, but visualization is just bad.

**Comment by la735681253** [comment]
> @ujgygy656565 Hello, what did you do to solve your problem? It appears that I have similar situation. While training, my RMSE is very small for both train and valid set, but visualization is just bad.

hi，when i have download the flownets_EPE1.951.pth.tar，it shown that The compressed file format is unknown or the data is corrupted，what is the problem？

**Comment by ghost** [comment]
> > try flownets_EPE1.951.pth.tar. Also it seems that your NN is undertrained.
> 
> Thank you. I've got the normal trajectory now. How to calculate the evaluation index of KITTI SLAM?

hello, Have you solved the problem? I also don't know how to calculate the evaluation index of KITTI SLAM

**Comment by ahaolaibiancheng** [comment]
> > try flownets_EPE1.951.pth.tar. Also it seems that your NN is undertrained.
> 
> Thank you. I've got the normal trajectory now. How to calculate the evaluation index of KITTI SLAM?
hello,could you tell me how to fix the problem you meet.Thx

**Comment by ahaolaibiancheng** [comment]
> > try flownets_EPE1.951.pth.tar. Also it seems that your NN is undertrained.
> 
> Thank you. I've got the normal trajectory now. How to calculate the evaluation index of KITTI SLAM?
hello,could you tell me how to fix the problem you meet.Thx@ujgygy656565
